pwnagotchi/plugins/default/quickdic.py

```diff
@@ -0,0 +1,51 @@
+__author__ = 'pwnagotchi [at] rossmarks [dot] uk'
+__version__ = '1.0.0'
+__name__ = 'quickdic'
+__license__ = 'GPL3'
+__description__ = 'Run a quick dictionary scan against captured handshakes'
+
+'''
+Aircrack-ng needed, to install:
+> apt-get install aircrack-ng
+Upload wordlist files in .txt format to folder in config file (Default: /opt/wordlists/)
+'''
+
+import logging
+import subprocess
+import string
+import re
+
+OPTIONS = dict()
+
+def on_loaded():
+    logging.info("Quick dictionary check plugin loaded")
+
+def on_handshake(agent, filename, access_point, client_station):
+    display = agent._view
+
+    result = subprocess.run(('/usr/bin/aircrack-ng '+ filename +' | grep "1 handshake" | awk \'{print $2}\''),shell=True, stdout=subprocess.PIPE)
+    result = result.stdout.decode('utf-8').translate({ord(c) :None for c in string.whitespace})
+    if not result:
+        logging.info("[quickdic] No handshake")
+    else:
+        logging.info("[quickdic] Handshake confirmed")
+        result2 = subprocess.run(('aircrack-ng -w '+OPTIONS['wordlist_folder']+'*.txt -l '+filename+'.cracked -q -b '+result+' '+filename+' | grep KEY'),shell=True,stdout=subprocess.PIPE)
```

Comment:
```
       -w <words>
              Path to a dictionary file for wpa cracking. Separate filenames with comma when using  multiple  dic‐
              tionaries.   Specify  "-"  to  use  stdin. [...]
```
This is going to break if `wordlist_folder` has more than one `.txt`.

Reply:
@kpcyrd hey dude why don't you send a PR and improve things yourself instead of being so critical with the work of others?

Reply:
no it doesn't... works fine with multiple .txt files, personally tested.

Reply:
I don't intend to be overly critical, I just don't have the time to develop and properly test a fix so I left a note instead.

Having multiple txt files kind of works, but only the first file is going to be used as a dictionary, the other files are going to be used as a pcap:

```
% touch empty.pcap
% echo foo > a.txt
% echo bar > b.txt
% # this works (ignoring the fact that my pcap is empty)
% aircrack-ng -w a.txt -q -b de:ad:be:ee:ee:ef empty.pcap
read(file header) failed: Success
0 potential targets

No matching network found - check your bssid.
% # this tries to open the 2nd dictionary as pcap
% aircrack-ng -w *.txt -q -b de:ad:be:ee:ee:ef empty.pcap
Unsupported file format (not a pcap or IVs file).
read(file header) failed: Success
0 potential targets

No matching network found - check your bssid.
```